src/Domain.LinnApps/Reports/PurchaseOrdersReportService.cs

```diff
@@ -390,61 +416,55 @@ private static void AddPartReportColumns(SimpleGridLayout reportLayout)
             ICollection<CalculationValueModel> values,
             PurchaseOrderDetail orderDetail,
             PurchaseOrderDelivery delivery,
-            decimal ledgerQty,
             string currencyCode,
             bool isFirstDelivery)
         {
             var currentRowId = $"{orderDetail.OrderNumber}/{orderDetail.Line}/{delivery.DeliverySeq}";
-            values.Add(
-                new CalculationValueModel
-                    {
-                        RowId = currentRowId,
-                        ColumnId = "OrderLine",
-                        TextDisplay = $"{orderDetail.OrderNumber}/Line{orderDetail.Line}/Delivery{delivery.DeliverySeq}"
-                    });
-
-            values.Add(
-                new CalculationValueModel
-                    {
-                        RowId = currentRowId, ColumnId = "PartNo", TextDisplay = $"{orderDetail.Part.PartNumber}"
-                    });
-
-            values.Add(
-                new CalculationValueModel
-                    {
-                        RowId = currentRowId,
-                        ColumnId = "SuppliersDesignation",
-                        TextDisplay = $"{orderDetail.SuppliersDesignation}"
-                    });
-
-            values.Add(
-                new CalculationValueModel
-                    {
-                        RowId = currentRowId, ColumnId = "QtyRec", TextDisplay = delivery.QtyNetReceived.ToString()
-                    });
-
-            values.Add(
-                new CalculationValueModel
-                    {
-                        RowId = currentRowId, ColumnId = "QtyInv", TextDisplay = ledgerQty.ToString(CultureInfo.InvariantCulture)
-                    });
-
-            values.Add(
-                new CalculationValueModel
-                    {
-                        RowId = currentRowId,
-                        ColumnId = "Currency",
-                        TextDisplay = currencyCode
-                    });
 
             if (isFirstDelivery)
             {
+                values.Add(
+                    new CalculationValueModel
+                        {
+                            RowId = currentRowId,
+                            ColumnId = "Order",
+                            TextDisplay = $"{orderDetail.OrderNumber}"
```

Comment:
Will moving these inside this if() not only show values for the first delivery now? Previously was separate to show all deliveries, but only show qty fields for further deliveries on the same detail, so that the totals didn't include duplicate values like 874003 here for example 
http://app.linn.co.uk/purchasing/reports/orders-by-part/report?partNumber=RAW%20251&fromDate=2022-05-01T10:55:16.000Z&toDate=2022-08-31T10:55:16.536Z&cancelled=N

Reply:
Ignore this actually, think was just github's weird formatting, the whole file makes more sense